src/BigQuery/Job.php

```diff
@@ -121,7 +127,8 @@ public function cancel(array $options = [])
     }
```

Comment:
Having a separate wait method is similar to what we are doing in Python, not that that is super important.

I do think there is value in making `queryResults` reflect the base API more closely, where it only makes a single call, but you are right that most everything is already on the `Job` object.